cloudy_mozdef/cloudformation/mozdef-instance.yml

```diff
@@ -136,7 +140,86 @@ Resources:
                 cookiename=sesmeteor
                 # Increase the AWS ES total fields limit from 1000 to 4000
                 OPTIONS_MAPPING_TOTAL_FIELDS_LIMIT=4000
+                # Set thresholds for attack dataviz lower means more ogres
+                OPTIONS_IPV4ATTACKERHITCOUNT=5
+                OPTIONS_IPV4ATTACKERPREFIXLENGTH=24
               path: /opt/mozdef/docker/compose/cloudy_mozdef.env
+            - content: |
+                #!/usr/bin/env python
+                # This Source Code Form is subject to the terms of the Mozilla Public
+                # License, v. 2.0. If a copy of the MPL was not distributed with this
+                # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+                # Copyright (c) 2014 Mozilla Corporation
+
+                from celery.schedules import crontab, timedelta
+                import time
+                import logging
+
+                ALERTS = {
+                    'bruteforce_ssh.AlertBruteforceSsh': {'schedule': crontab(minute='*/1')},
+                    'unauth_ssh.AlertUnauthSSH': {'schedule': crontab(minute='*/1')},
+                    'guard_duty_probe.AlertGuardDutyProbe': {'schedule': crontab(minute='*/1')},
+                    'cloudtrail_logging_disabled.AlertCloudtrailLoggingDisabled': {'schedule': timedelta(minutes=1)},
+                    'cloudtrail_deadman.AlertCloudtrailDeadman': {'schedule': timedelta(hours=1)}
+                }
+
+                ALERT_PLUGINS = [
+                    # 'relative pythonfile name (exclude the .py) - EX: sso_dashboard',
+                ]
+
+                RABBITMQ = {
+                    'mqserver': 'rabbitmq',
+                    'mquser': 'guest',
+                    'mqpassword': 'guest',
+                    'mqport': 5672,
+                    'alertexchange': 'alerts',
+                    'alertqueue': 'mozdef.alert'
+                }
+
+                ES = {
+                    'servers': ["${ESURL}"]
```

Comment:
Is it not possible to use the existing `/opt/mozdef/docker/compose/cloudy_mozdef.env` pathway which makes the `ESURL` value available instead of creating config.py?

For example instead of creating `config.py` here in the user-data of the instance, change the `config.py` on disk to something like

```
ES = {
    'servers': [os.environ.get('OPTIONS_ESURL')]
```

Reply:
We also need to template out that file to activate different alerts.  I'd suggest merging this as is and filing an issue to change the way this works entirely to use a different config system.